contrib/tracing/log_coins_flushed.py

```diff
@@ -45,12 +49,26 @@
     'ALWAYS'
 ]
 
+# define output data structure in Python
+class Data(ct.Structure):
+    _fields_ = [
+        ("ts", ct.c_long),
```

Comment:
Could this be a `c_uint64` too?
```suggestion
        ("ts", ct.c_uint64),
```

Reply:
Yup, there was a type `c_uint64` and it seemed to work without any issue